am-lib/src/main/java/uk/gov/hmcts/reform/amlib/DummyService.java

```diff
@@ -1,8 +1,33 @@
 package uk.gov.hmcts.reform.amlib;
 
+import org.jdbi.v3.core.Jdbi;
+
+import java.util.List;
+
 public class DummyService {
+    private Jdbi jdbi;
+    private String msg = "Hello Dummy Service 2";
+
+    public DummyService(String dbUrl, String user, String password) {
```

Comment:
It would be good to add another constructor with usage of javax.sql.DataSource;
so that in the service there would be two things:
Config class with:
```    @Bean
    @ConfigurationProperties("spring.datasource")
    public DataSource dataSource() {
        return DataSourceBuilder.create().build();
    }
```

and usage as `@Autowired DataSource ds;`

`Jdbi.create(ds);` <- which btw could be put into a separate class/Bean config with PostgresPlugin attached/installed.

Reply:
I will use this pattern in am-api

Reply:
would you like to add this code in seperate PR and bump with new version as I've already  released 0.0.5 version ?